README.md

```diff
@@ -102,6 +102,9 @@ $ src/repology/repology_cve.py --help
 
 # nix_outdated:
 $ src/nixupdate/nix_outdated.py --help
+
+# provenance:
+$ src/provenance/main.py --help
```

Comment:
You could also add a link to the provenance.md on this README among the other listed tools section: https://github.com/tiiuae/sbomnix/blob/main/README.md?plain=1#L10-L15